direct/src/extensions_native/NodePath_extensions.py

```diff
@@ -200,7 +218,8 @@ def pPrintString(self, other = None):
 
 def printPos(self, other = None, sd = 2):
         """ Deprecated.  Pretty print a node path's pos """
-        print("NodePath.printPos() is deprecated.")
+        if __debug__:
+                warnings.warn("NodePath.printPos() is deprecated.", DeprecationWarning, stacklevel=2)
```

Comment:
This line and other warnings lines in this file are indented too much (I don't know why the functions themselves are also indented too much, but let's use 4 space indents for these)

Reply:
Hmm. My editor is set to 4 spaces per tab. Looking at the file, it appears there were 8 spaces for each one. I will fix them all.

Reply:
Fixed it. It was very strange. Not sure when this happened.

Reply:
Yeah, weird. Fixed.